crabml-core/src/backends/cpu/primitives/batch_matmul_vec.rs

```diff
@@ -106,7 +105,7 @@ fn batch_matmul_vec_f16(
 ) {
     if a_stride2 == 1 {
         let _t = device.metrics.batch_matmul_rowwise_walltime.track();
-        cbuf.par_iter_mut().enumerate().for_each(|(i, bufcp)| {
+        cbuf.iter_mut().enumerate().for_each(|(i, bufcp)| {
```

Comment:
i found that replacing par_iter_mut() to `iter_mut` here is helpful for calculating attentions whose context window is not that long (like below <100), imho this change could be kept. (batch_matmul_vec is only called on calculating attentions, in which the calcuation grows as the tokens get longer)

but the matmul_vec.rs side, parallelism is still a must on caculating big GEMV.